### ISSUE TITLE
[E&A] Preselection of presets and hand-drawn polygons across page reloads

### ISSUE THREAD

**Issue by dzole0311** [NASA-IMPACT/veda-ui#950]
1. When a preset is selected and the page is reloaded, the preset option is not shown as selected. We should probably keep this selection as a url parameter ("preset=Alaska") and then load it in the PresetSelector, so that the user knows which state was selected. 
2. Additionally, we should highlight the selected hand-drawn geometries on the map across page reloads. If I draw 3 polygons, select 2 of them and reload the page, they are not again selected due to diverging states between MapboxDraw and our React atoms. We already keep the state of the selected aois in the url, so we should switch those to "simple_select" mode so that they can be automatically highlighted
3. It might be a good idea to differentiate the colors of the geometries based on if they are "static" (presets, file uploads) or hand-drawn (@faustoperez any thoughts?)

**Comment by faustoperez** [comment]
> It might be a good idea to differentiate the colors of the geometries based on if they are "static" (presets, file uploads) or hand-drawn (@faustoperez any thoughts?)

To simplify the work I suggest (for now) that they be differentiated by the presence of the edit handles, but use the same "this is a shape" stroke and fill color 👍

**Comment by AliceR** [comment]
Hi @dzole0311 ! Thanks for outlining these issues! Here’s how the new proposed user flow (#1395)  addresses points 2 and 3, along with some thoughts on the first issue:

2. Highlighting selected hand-drawn geometries across reloads:
The new approach simplifies AOI management by always having a single active AOI. Since multiple selectable polygons are no longer supported, the issue of diverging states between MapboxDraw and our React atoms is resolved. Any drawn geometry is (after confirmation) treated as the active AOI, synced with the URL, and reloaded automatically as part of the map state.

3. Differentiating geometry colors based on source:
Since all AOIs—whether presets, file uploads, or hand-drawn—are now treated equivalently in the new workflow, it wouldn’t make sense to differentiate them by color. All AOIs are equal in terms of functionality and use in the analysis. Adding a visual distinction could introduce unnecessary complexity or confusion. The consistent treatment of AOIs ensures a simpler and more streamlined user experience.

---

1. Dropdown indicating selected preset upon reload
This one is worth discussing further. Once a preset is selected, it essentially becomes "just an AOI" and is treated the same way as any other AOI, with no dynamic link to the preset. Because of this, I’d suggest not persisting the dropdown selection upon reload. Doing so might confuse users by implying that the preset remains active as something separate from the AOI.

Instead, the map state should clearly reflect the selected AOI. This approach keeps the logic simple and avoids introducing unnecessary dependencies between the dropdown and the AOI.

Let me know what you think about this! If we do see value in keeping the preset selection tied to the dropdown, we could explore alternatives—but keeping it decoupled seems more straightforward for now.

**Comment by aboydnw** [comment]
I agree on 2 & 3, I think those are effectively resolved with #1395 

For the first point, I do think there is some improvement in the future here, because when you select an AOI there is an x that allows you to deselect the AOI. This is duplicative functionality with the Delete Area button at the bottom of the map. 

So, we could probably simplify somehow, and I think it could make sense to keep the dropdown selection in this case.

However, I think that #1395 represents a large enough simplification of the workflow that the main pain point behind this ticket is addressed, so I would propose closing this alongside #1395 and opening a new ticket if we think there is more confusion on the first point above.

If you all agree, I'll leave in In Review until #1395 is closed

![Image](https://github.com/user-attachments/assets/930af0f3-0794-4fa9-8f3d-95a48e9080d2)
